cg-producer/setup.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+def get_long_desc():
+    with open("pypi-readme.md", "r") as readme:
```

Comment:
Why not just name this `README.md`?

Reply:
The reason behind this additional file is that we have a main [Readme file](https://github.com/fasten-project/pypi-tools/blob/refactor-pycg-producer/cg-producer/README.md) which  includes instructions on running the Kafka Call Graph Generator. But we wanted to create an additional description file for the  pycg-producer library, which is distributed through PyPI and needed a description.